hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/BatchStage.java

```diff
@@ -66,6 +67,23 @@
     @Nonnull @Override
     BatchStage<T> rebalance();
 
+    /**
```

Comment:
Public APIs should include a detailed explanation, sample usage, and parameter and return value descriptions. Could you add them? You can refer to the methods in the `GeneralStage` interface.